configure.ac

```diff
@@ -39,6 +39,8 @@ AC_CHECK_HEADER(uuid/uuid.h, [],
 		[AC_MSG_ERROR([Couldn't find uuid/uuid.h])])
 AC_CHECK_HEADER(libdevmapper.h, [],
 		[AC_MSG_ERROR([Couldn't find libdevmapper.h])])
+AC_CHECK_HEADER(kmod/libkmod.h, [],
```

Comment:
On my Fedora box, kmod header is at /usr/include/libkmod.h, not /usr/include/kmod/libkmod.h... I guess you are using SLES ? Does SLES change the install path for kmod-devel ?